src/docbuilder/rustwide_builder.rs

```diff
@@ -516,16 +516,24 @@ impl RustwideBuilder {
         let mut storage = LogStorage::new(LevelFilter::Info);
         storage.set_max_size(limits.max_log_size());
 
+        // we have to run coverage before the doc-build because currently it
+        // deletes the doc-target folder.
+        // https://github.com/rust-lang/cargo/issues/9447
+        let doc_coverage = match self.get_coverage(target, build, metadata, limits) {
```

Comment:
Do you mind adding a build test that coverage is generated? Maybe it could look at the /crate page and see if there's a % sign somewhere (see https://docs.rs/crate/tokio/1.5.0 for an example of what that looks like).

Reply:
I'll add a test. 
Instead of checking the HTML output I would propose we just check if the coverage is in the database, which is good enough to know if the build with coverage worked, right?

Reply:
Yes, that's a better test :)

Reply:
@jyn514 I added the assertion to the build-test